src/API/Site/Controllers/CountryCodeTrait.php

```diff
@@ -35,24 +36,29 @@ protected function validate_country_code( string $country ): void {
 	}
 
 	/**
-	 * Validate that a country or a list of countries is valid and supported.
+	 * Validate that a country or a list of countries is valid and supported,
+	 * and also validate the data by the built-in validation of WP REST API with parameter’s schema.
 	 *
-	 * @param mixed $countries                An individual string or an array of strings.
-	 * @param bool  $check_supported_country  Whether to check the country is supported.
+	 * @param bool    $check_supported_country  Whether to check the country is supported.
+	 * @param mixed   $countries                An individual string or an array of strings.
+	 * @param Request $request                  The request to validate.
+	 * @param string  $param                    The parameter name, used in error messages.
```

Comment:
Maybe we can add somewhere a short explanation of your findings of why `rest_validate_request_arg` is not being called automatically.

Reply:
Added in https://github.com/woocommerce/google-listings-and-ads/pull/2003/commits/4eeffe896c0dc87a8ddeff94e2941e1eabad6f28.